CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/AudioCard.cs

```diff
@@ -73,13 +73,13 @@ public AudioCard(string title = default(string), string subtitle = default(strin
         /// Array of media Url objects
         /// </summary>
         [JsonProperty(PropertyName = "media")]
-        public IList<MediaUrl> Media { get; set; }
+        public IList<MediaUrl> Media { get; set; } = new MediaUrl[0];
```

Comment:
which is the intent. The problem is when devs do `new()` and `.Add` on the `Media` property, it blows up because `.Media` is `null`